nuxeo-coldstorage/src/test/java/org/nuxeo/coldstorage/S3TestHelper.java

```diff
@@ -0,0 +1,212 @@
+                                                                      key.getKey()))
+                                                              .collect(Collectors.toList());
+        if (keys.size() > 0) {
+            DeleteObjectsRequest multiObjectDeleteRequest = new DeleteObjectsRequest(bucketName).withKeys(keys)
+                                                                                                .withQuiet(false);
+            DeleteObjectsResult deleteObjectsResult = amazonS3.deleteObjects(multiObjectDeleteRequest);
+            int successfulDeletes = deleteObjectsResult.getDeletedObjects().size();
+            log.info(successfulDeletes + " objects successfully deleted.");
+        }
+    }
+
+    public AmazonS3 getAmazonS3(String provider) {
+        S3BlobProvider s3BlobProvider = (S3BlobProvider) Framework.getService(BlobManager.class)
+                                                                  .getBlobProvider(provider);
+        return s3BlobProvider.config.amazonS3;
+    }
+
+    public String getMainBucket() {
+        return mainBucket;
+    }
+
+    public String getGlacierBucket() {
+        return glacierBucket;
+    }
+
+    public String getRegion() {
+        return region;
+    }
+
+    public String getBucketPrefix() {
+        return bucketPrefix;
+    }
+
+    protected static String getEnvValue(String key) {
+        return StringUtils.trim(System.getenv(key));
+    }
+
+    /**
+     * Update object class storage associated with the document of the given {@link DocumentModel}. after moving a new
+     * document into Coldstorage Bucket, the storage class update will be handled by an S3 Lifecycle management
+     * https://docs.aws.amazon.com/AmazonS3/latest/userguide/lifecycle-transition-general-considerations.html but when
+     * running the unit tests we can't wait the storage transition event to happen ( more than 24h)
+     *
+     * @param document the document
+     */
+    public void updateObjectClassStorageToGlacier(DocumentModel document) {
+        AmazonS3 amazonS3 = this.getAmazonS3("glacier");
+        String blobkey = this.bucketPrefix + "/"
+                + this.getBlobKey(document, ColdStorageHelper.COLD_STORAGE_CONTENT_PROPERTY);
+        log.info("ColdStorage Blob key {}", blobkey);
+        CopyObjectRequest copyObjectRequest = new CopyObjectRequest(this.glacierBucket, blobkey,
+                this.getGlacierBucket(), blobkey).withStorageClass(StorageClass.Glacier);
+        if (amazonS3.doesObjectExist(this.glacierBucket, blobkey)) {
+            amazonS3.copyObject(copyObjectRequest);
+        } else {
+            log.info("ColdStorage Blob key {} doesn't exist", blobkey);
+        }
+
+    }
+
+    /**
+     * Check If an S3 object is being retrieved.
+     *
+     * @param document the document
+     * @return the boolean
+     */
+    public boolean ifS3ObjectBeingRetrieved(DocumentModel document) {
```

Comment:
`is` instead `if`. 

you could also simplify  the java doc by using only the return `@return {@code true} if the blob's document is being retrieved from {@code Glacier} storage, {@code false} otherwise`